### PR TITLE
this branch is for review

### PR DESCRIPTION
**Pull request by zuhakhan028**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (anirudhramanan) [inline .vscode/settings.json]: this is not needed

File: index.html [hunk 2]
```diff
@@ -180,8 +192,61 @@ <h3 class="price">Rs 44,999</h3>
...
+
+           </div>
+
+           <div class="qna">
+            <ul class="qna_ul">
+               <div id="close_functionality3"><li class="que1"><h3>How does it work?</h3><div id="toggle3"><img class="cross_img" src="images/arrow-down.png"></div></li></div>
+                 </ul>
+
+           </div>
+
```
Comment (anirudhramanan) [inline index.html:249]: lot of empty spaces, please remove

File: index.html [hunk 3]
```diff
@@ -180,8 +192,61 @@ <h3 class="price">Rs 44,999</h3>
          </ul>
     
             </div>
+            <input  class="button" type="submit" name="" id="">
            </div> 
-           
+
```
Comment (anirudhramanan) [inline index.html]: lot of empty spaces, please remove

File: index.js [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+var close_fnc3 = document.getElementById("close_functionality3");
+var togglebtn3 = document.getElementById("toggle3");
+togglebtn3.addEventListener("click", function () {
+  var docdiv = document.createElement("div");
+  docdiv.innerHTML = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+  docdiv.classList.add("innerHTML")
+  docdiv.classList.add("togglediv")
+  close_fnc3.appendChild(docdiv);
+})
+
```
Comment (anirudhramanan) [inline index.js:45]: lot of empty spaces, please remove

File: index.js [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+}
+
+function rotate2() {
+  transform2.classList.add("cross_img_rotate")
+}
+
+function rotate3() {
+  transform3.classList.add("cross_img_rotate")
+}
+
```
Comment (anirudhramanan) [inline index.js:80]: lot of empty spaces, please remove

**Comment by anirudhramanan** [comment]
LGTM

**Comment by zuhakhan028** [comment]
@anirudhramanan all comments resolved.

**Pull request merged**